src/SourceBrowser/src/Directory.Packages.props

```diff
@@ -12,6 +12,9 @@
     <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.5.0" />
     <PackageVersion Include="Microsoft.Build" Version="17.5.0" />
 
+    <!-- component governance -->

```

Comment:
Not sure it's helpful to say `component governance` here.  We need this because it's a transitive dependency of MSBuild.

Reply:
Oh, sorry, had auto-merge set so it merged after your approval but I can fix this in a follow up. The thinking here is that this isn't a direct dependency, so I wanted to add a section for Versions that are only there due to lifting transitive dependencies for CG. We do that in Aspire too. This way, we know we can safely remove those once we update direct dependencies that have bumped the transitive version.